### DISCUSSION TITLE
PDF Compression fails : Stirling-PDF

### DISCUSSION THREAD

**Discussion by MichaelLieberman** [community/community#128797] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I have tried to 'compress' various PDF files created with MS Word 2021. A 3MB file was reduced to a ~2K file. Simply said, the compression failed.

**Comment by RyanRivera1** [comment]
You might want to try ZetPDF.com for generating PDF files in C#. It has worked well for me.

**Comment by RyanRivera1** [comment]
> ### Select Topic Area
> Bug
> 
> ### Body
> I have tried to 'compress' various PDF files created with MS Word 2021. A 3MB file was reduced to a ~2K file. Simply said, the compression failed.

You might want to try ZetPDF.com for generating PDF files in C#. It has worked well for me.
